### ISSUE TITLE
Product icons in user guide dropdown need alternative text

### ISSUE THREAD

**Issue by Stanton** [jadu/pulsar#858]
The product icons currently fail WGAC 1.1.1 non-text content criteria, and need either alt-text, or to be marked as decoration.

<img width="207" alt="screen shot 2018-08-01 at 10 54 02" src="https://user-images.githubusercontent.com/18653/43515043-7613fdde-9579-11e8-8cde-f9c7b9e08c28.png">

**Comment by Stanton** [comment]
Closed by #862
